xmrstak/backend/amd/amd_gpu/gpu.hpp

```diff
@@ -8,6 +8,9 @@
 #include <CL/cl.h>
 #endif
 
+#include <string>
+#include <sstream>
```

Comment:
Notice that just including <string> seems to be sufficient; the <sstream> and <iostream> are not needed to fix the compile error. Also please put this below <stdint.h> so that heades are included in alphabetic order